### PR TITLE
Allow line terminators in @font-face declaration

### PR DESCRIPTION
**Pull request by stevenbfine**
This pull request is in response to @progers comment that it didn't work on one of the sites he tried (http://dglazkov.com). It was a small fix (line 407), and I changed the tests to test for this scenario. However, I also made a few changes that handle some edge cases that weren't previously handled.

### CODE REVIEW

File: tests/tests.js [hunk 1]
```diff
@@ -464,29 +464,34 @@ QUnit.test('fullyQualifiedFontURL', function(assert) {
...
   var serializer = new HTMLSerializer();
   var cssText = 'body{color:red;}' +
-      '@font-face{font-family:Font;src:url("/hello/")}';
+      '@font-face{font-family:Font;\nsrc:url("/hello/")}';
   var href = 'http://www.example.com/';
   serializer.processCSSFonts(window, href, cssText);
   assert.equal(
     serializer.fontCSS[0],
-    '@font-face{font-family:Font;src:url("http://www.example.com/hello/")}'
+    '@font-face{font-family:Font;\nsrc:url("http://www.example.com/hello/")}'
```
Comment (chrishtr) [inline tests/tests.js]: Make new examples rather than modifying existing ones.

Comment (stevenbfine) [inline tests/tests.js]: Done

**Comment by chrishtr** [comment]
LGTM
